Neos.Neos/Classes/Fusion/Helper/ArrayHelper.php

```diff
@@ -90,6 +91,14 @@ public function groupBy($set, $groupingKey)
         return $result;
     }
 
+    /**
+     * Sorts the input array by the $positionProperty of each element.
+     */
+    public function sortByPosition(array $set, $positionPropertyPath = 'position'): array
```

Comment:
I would suggest to call this one `sortByKey` or `sortBy` instead:
```suggestion
    public function sortByKey(array $set, $positionPropertyPath = 'position'): array
```

Reply:
Or rather `sortByPropertyPath`? 
Key would be incorrect.

Reply:
yes, also fine with me even though it's not really a property

Reply:
```suggestion
    public function sortByPropertyPath(array $set, $positionPropertyPath = 'position'): array
```

Reply:
Made the adjustment